packages/sdk/src/swap/types.ts

```diff
@@ -129,7 +129,8 @@ type SwapState =
   | {
       state: 'AWAITING_DEPOSIT';
       depositAmount: string | undefined;
-      depositTransactionHash: string | undefined;
+      depositTransactionHash: string | undefined; // DEPRECATED(1.5): use depositTransactionRef instead
```

Comment:
unfortuante existing naming bc the polkadot thing is not a hash (and we call it `ref` for broadcasts). will expose both names for now